modules/ROOT/pages/dataweave-formats-dw.adoc

```diff
@@ -1,103 +1,64 @@
-= DataWeave Format (dw)
-// :page-aliases: 4.3@mule-runtime::dataweave-formats-dw.adoc
 
-MIME Type: `application/dw`
+= Dw Format
```

Comment:
TODO: "Dw" (can't use module.label() in this case). Can we do something special for this case and any others.

Reply:
DW-759